src/Resources/tests/unit/api/putEventUpdate.spec.ts

```diff
@@ -0,0 +1,36 @@
+import useApi from '@/api/api';
```

Comment:
inkonsistent, hier single quotes, in anderen dateien double quotes

Reply:
wurde durch double quotes ersetzt